src/frontend/src/hooks/finance.hooks.ts

```diff
@@ -260,6 +261,28 @@ export const useApproveReimbursementRequest = (id: string) => {
   );
 };
 
+/**
+ * Custom react hook to deny a reimbursement request for the finance team
+ *
+ * @param id id of the reimbursement request to deny
+ * @returns the denied reimbursement request
```

Comment:
When we look at our RR deny endpoint in the RR services file, we can see that the RR deny function returns RR status instead of RR. Let's fix this @returns comment